diff --git a/src/adapters/pandoc.rs b/src/adapters/pandoc.rs
index 0eaf7ed..7382b7c 100644
--- a/src/adapters/pandoc.rs
+++ b/src/adapters/pandoc.rs
@@ -80,7 +80,7 @@ impl SpawningFileAdapter for PandocAdapter {
             //.arg("--to=commonmark-header_attributes-link_attributes-fenced_divs-markdown_in_html_blocks-raw_html-native_divs-native_spans-bracketed_spans")
             .arg("--to=plain")
             .arg("--wrap=none")
-            .arg("--atx-headers");
+            .arg("--markdown-headings=atx");
         cmd
     }
 }
